vault/secret.go

```diff
@@ -36,3 +37,33 @@ func (v Vault) Secret(path string) (*Secret, error) {
 	}
 	return secret, nil
 }
+
+// DeleteSecret deletes the secret at path from the DSV of the given tenant.
+// See https://dsv.secretsvaultcloud.com/api#operation/deleteSecret.
+func (v Vault) DeleteSecret(path string) error {
+	_, err := v.accessResource(http.MethodDelete, secretsResource, path, nil)
+	return err
+}
+
+// SecretRequest represents the request body of the CreateSecret operation.
+type SecretRequest struct {
+	Attributes map[string]interface{}
+	Data       map[string]interface{}
+	Description string
+}
+
+// CreateSecret creates the secret at path in the DSV of the given tenant.
+// See https://dsv.secretsvaultcloud.com/api#operation/createSecret.
+func (v Vault) CreateSecret(path string, req SecretRequest) (*Secret, error) {
```

Comment:
please use a pointer to the `SecretRequest` type

Reply:
Updated in 09796e9.